plugins/inputs/win_wmi/README.md

```diff
@@ -0,0 +1,280 @@
+# Windows Management Instrumentation Input Plugin
+
+This document presents the input plugin to read WMI classes on Windows
+operating systems. With the win_wmi plugin, an administrator is enabled to
+capture and filter virtually any configuration or metric value exposed through
+the Windows Management Instrumentation
+([WMI](https://learn.microsoft.com/en-us/windows/win32/wmisdk/wmi-start-page))
+service.
+

```

Comment:
You mention an administrator, can we document that in troubleshooting please?

Reply:
In this context, I was referring to an operator or IT professional rather than a user with administrative access. WMI is protected by ACLs, so any user with read access to a namespace or class will be able to query a class. Otherwise, an access denied error will be presented to the operator. I'll fix the documentation to be more clear.

Reply:
Ah ok, then yeah two changes:

* Remove the verbiage of administrator because "administrator" in windows context is easy to understand as "administrator privileges"
* add a comment about the namespace/ACL permissions please

Thanks!

Reply:
Done.